_conferences/big-android-bbq-amsterdam-2015.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "Big Android BBQ Amsterdam"
```

Comment:
You don't need the location in the name. We will always render the location with the name.